crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -215,4 +216,18 @@ def upgrade
       end
     end
   end
+
+  def check_eol
+    Rails.cache.write(:eol_checked, true, expires_in: 24.hours)
+
+    branding = YAML.load_file(Rails.root.join("config", "branding.yml"))
+    return if branding["eol_date"].blank?
+    return if Time.parse(branding["eol_date"]).future?
```

Comment:
I found nothing in the channels but we could use the `/etc/products.d/suse-openstack-cloud.prod` file.